politeiawww/proposals.go

```diff
@@ -202,46 +202,47 @@ func validateProposal(np www.NewProposal, u *user.User) error {
 	// Check for at least 1 markdown file with a non-empty payload.
 	if len(np.Files) == 0 || np.Files[0].Payload == "" {
 		return www.UserError{
-			ErrorCode: www.ErrorStatusProposalMissingFiles,
+			ErrorCode: www.ErrorStatusNoIndexFile,
```

Comment:
You can't just switch the error message out here.  It doesn't make sense.  You need to read through the entire validateProposal function and make sure everything makes sense in the context of the new policies.

Reply:
My thought here was, since this is checking for at least 1 non-empty markdown file, the error is inevitably that no index file exist in the proposal, since it is the only mandatory file for one. In line 257 a similar check exists, but since both of them exists in master, I thought best not to remove any.

Reply:
This is not checking for 1 non-empty markdown file though.  This is just checking for 1 non-empty file.

Reply:
My bad, sorry about that. Fixing this

Reply:
Fixed and corrected tests to match the changes